custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/src/main/java/com/datarobot/custom/ScoringCode.java

```diff
@@ -68,10 +52,12 @@ public String predict(String inputFilename) throws Exception {
                     csvPrinter.printRecord(value);
                 }
             } else {
-                csvPrinter = new CSVPrinter(new StringWriter(), CSVFormat.DEFAULT.withHeader(this.positiveClassLabel, this.negativeClassLabel));
+                var labelsArr = this.classLabels.toArray(new String[this.classLabels.size()]);
```

Comment:
nit: maybe we could use `String[]` instead of `List<String>` for `classLabels`, so that there's no `toArray` conversion.